client/src/modules/templates/modals/project.modal.html

```diff
@@ -41,6 +42,46 @@
       </div>
     </div>
 
+    <!-- project logo -->
+    <div class="form-group">
+      <div style="display: flex; flex-direction: column; justify-content: center; align-items:center; margin-bottom: 10px;">
```

Comment:
Instead of introducing custom `flex` styles, could you use bootstrap classes for these?  It will make it easier to upgrade in the future.

I think all you need are `row` classes with different `col-xs` and `col-md` classes.  I would imagine you could use the `text-alignment` classes for center the items (provided you used `<span>` and not `<div>`) ([link](https://getbootstrap.com/docs/3.4/css/#type-alignment)) or you might have better luck with `center-block` ([src](https://getbootstrap.com/docs/3.4/css/#images-responsive)).  Also, you could use `image-thumbnail` for the image ([link](https://getbootstrap.com/docs/3.4/css/#images-shapes)).